dbbackup/db/mysql.py

```diff
@@ -0,0 +1,36 @@
+from .base import BaseCommandDBConnector
+
+
+class MysqlDumpConnector(BaseCommandDBConnector):
+    """
+    MySQL connector, creates dump with ``mysqldump`` and restore with
+    ``mysql``.
+    """
+    dump_cmd = 'mysqldump'
+    restore_cmd = 'mysql'
+
+    def create_dump(self):
+        cmd = '%s %s --quick' % (self.dump_cmd, self.settings['NAME'])
```

Comment:
Could / should all these `%` string formattings perhaps use `.format` for modernization's sake :)